docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+app:
+  extends:
+    file: docker/common.yml
+    service: app
+  links:
+    - db
+  command: rackup -s puma -p 4567 --host 0.0.0.0
+  ports:
+    - "4567:4567"
```

Comment:
Hmm, so the reason that I set it to 4567 is because the rest of the dev setup seems to assume it's at that port too, as @mhelmetag noted. I guess I'd rather just keep it at 4567 by default to keep things consistent--however, I believe you can also forcibly override this by adding a [`docker-compose.override.yml`](https://docs.docker.com/compose/extends/) in the root directory with the following in it:

```yml
app:
  command: rackup -s puma -p 5000 --host 0.0.0.0
ports:
  - "5000:5000"
```

Hmm, come to think of it though, if we add a `PORT` env var to the `.env` file, we could potentially leverage [docker-compose's environment file](https://docs.docker.com/compose/env-file/) and use that variable in the `docker-compose.yml`. This would keep things nice and DRY and make overriding the port easier, since you could just change the `PORT` setting in your `.env` file instead of making a `docker-compose.override.yml`.